sda-svc/values.yaml

```diff
@@ -132,6 +132,8 @@ global:
 # name of the c4gh key file.
     file: c4gh.key
     passphrase: ""
+# name of the public file, required for testing
```

Comment:
maybe we should have something separated for test in each config like what is for deployment and what for test.

e.g. here we have `global` and we can have a `tests` key specific for that- but this is more of an opinion rather than a request and either work for me.